### DISCUSSION TITLE
Why do we calculate stft in VITS  loss? It never seems to used for loss propagation.

### DISCUSSION THREAD

**Discussion by p0p4k** [coqui-ai/TTS#2467] [General Q&A] [answered]
https://github.com/coqui-ai/TTS/blob/c2d15cd41335798bfa81bc9376bdd35e5e187590/TTS/tts/layers/losses.py#L623-L633

**Comment by erogol** [comment]
why do you think it is not being used?

**Reply by erogol** [reply]
Because it is not being used :)

**Comment by erogol** [comment, marked as answer]
Looks like it can be removed. Would you send a PR? Thanks for the catch.
